### PR TITLE
Fix update ss template FormActions

### PR DESCRIPTION
**Pull request by flamerohr**
To be closer to React's implementation

Fixes https://github.com/silverstripe/silverstripe-framework/issues/6439

Dependencies
https://github.com/silverstripe/silverstripe-cms/pull/1819
https://github.com/silverstripe/silverstripe-admin/pull/62

### CODE REVIEW

**Comment by tractorcow** [comment]
ButtonContent is raw HTML: Should we only add the span if using default $Title.XML field?

**Comment by flamerohr** [comment]
@tractorcow good point, just looking at the React `FormAction`, looks like that doesn't support anything but simple text for title/content.

I'll update both to be similar in behaviour

File: templates/SilverStripe/Forms/FormAction.ss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% if $UseButtonTag %>
 	<button $AttributesHTML>
-		<% if $ButtonContent %>$ButtonContent<% else %>$Title.XML<% end_if %>
+		<span><% if $ButtonContent %>$ButtonContent<% else %>$Title.XML<% end_if %></span>
```
Comment (tractorcow) [inline templates/SilverStripe/Forms/FormAction.ss]: (Updating comment to be review)

If we set html directly via $ButtonContent, then we shouldn't wrap that in a span.

**Comment by tractorcow** [comment]
> @tractorcow good point, just looking at the React FormAction, looks like that doesn't support anything but simple text for title/content.

We don't need to support react buttoncontent just yet, but we can't break non-react form actions.

**Comment by flamerohr** [comment]
@tractorcow I've updated the templates not wrap `$ButtonContent` in a `<span>` and React now uses `castStringToElement()` instead of hardcoded `<span>`

**Pull request merged**